src/Turing.jl

```diff
@@ -70,6 +71,7 @@ export  @model,                 # modelling
         @logpdf,
         @sampler,
         DynamicPPL,
+        mapvcat,
```

Comment:
Do we need actually have to reexport `mapvcat`?

Reply:
No we don't. I will remove it.